package.json

```diff
@@ -66,6 +66,7 @@
     "mdfind": "^1.0.0",
     "minimist": "^1.2.0",
     "mkdirp": "^0.5.1",
+    "moment": "^2.18.1",
```

Comment:
Moment is a pretty big package to drag in just to do date formatting. The portal is using `date-fns` instead specifically because moment is so big. Do we really need moment for what you're doing?

Also, does this package include typings already? There's not typings file added in the PR?